tests/fxl-orientation_default/OPS/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Fixed Layout</dc:coverage>
+    <dc:creator>Wendy Reid</dc:creator>
+    <dc:date>2021-11-18</dc:date>
+    <dc:description>Reading systems should display the EPUB in the orientation of the device</dc:description>
```

Comment:
Period at end, please.

Reply:
I think this needs to say "rendition:orientation is not specified. Therefore, it has the value "default" and reading systems..."